### PR TITLE
Automatically test README and scripts in examples/

### PR DESCRIPTION
**Pull request by andrrizzi**
This is an attempt to test with Travis every example in the `examples/` folder and in the `README.md` file in the root directory (see #211).

I still have to add testing for the ipython notebooks, but I want to check that Travis finds the example files first.

### CODE REVIEW

**Comment by andrrizzi** [comment]
It seems to work! I'll add the support for notebooks next.

File: openforcefield/typing/chemistry/environment.py [hunk 1]
```diff
@@ -84,11 +84,11 @@ def _convert_embedded_SMIRKS(smirks):
     a_out = 0
     while smirks.find('$(') != -1:
         # Find first atom
-        atom, a_in, a_out = _find_embedded_brackets(smirks, '\[', '\]')
+        atom, a_in, a_out = _find_embedded_brackets(smirks, r'\[', r'\]')
```
Comment (j-wags) [inline openforcefield/typing/chemistry/environment.py]: Huzzah! Thanks for cleaning up these warnings.

**Comment by andrrizzi** [comment]
Great, thanks!

**Comment by davidlmobley** [comment]
Thanks!

**Pull request merged**